wire-ios/Wire-iOS/Sources/UserInterface/Settings/Debug Report/SettingsDebugReportViewController.swift

```diff
@@ -0,0 +1,168 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import MessageUI
+import UIKit
+import WireDataModel
+import WireDesign
+
+class SettingsDebugReportViewController: UIViewController {
+
+    // MARK: - Constants
+
+    enum LayoutConstants {
+        static let spacing: CGFloat = 8
+        static let padding: CGFloat = 20
+        static let safeBottomPadding: CGFloat = 30
+        static let buttonHeight: CGFloat = 48
+    }
```

Comment:
We can make this private:

```suggestion
    private enum LayoutConstants {
        static let spacing: CGFloat = 8
        static let padding: CGFloat = 20
        static let safeBottomPadding: CGFloat = 30
        static let buttonHeight: CGFloat = 48
    }
```

Reply:
https://github.com/wireapp/wire-ios/pull/1658/commits/c81926da4e3baa2bf5253ca1bc051109d41a4c71